### ISSUE TITLE
Don’t know whether to report this under bug or feature, but I’d love to see accessibility get improved.

### ISSUE THREAD

**Issue by CreepyWaters** [Trustroots/trustroots#2654]
I have used both the website and the app, and I noticed some issues with accessibility. First off the icons for navigation are not labeled to be screen reader compatible. Secondly, the map for finding members and/or finding a host or both in accessible. I appreciate the search bar in the map but once you put in the place you were looking into there is no way to get a plain text list  of profiles. I am partially blind, and cannot see maps and rely on my screen reader to read text to me. I would love to see this get fixed. Thank you so very much!  the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

Add links to discussion on Slack (or Discourse).

**Comment by mrkvon** [comment]
It's excellent to have this feedback. Thank you @CreepyWaters for pointing it out and sharing your experience!

I noticed there is a missing aria label at least at the main profile menu, but i'm guessing there is more. This could be low-hanging fruit.

Regarding the map, maybe the list of results could be displayed in the "Results" tab, and the results that are closest to the map center could be listed first. This may be a bit more complex to implement, but definitely a big step for the map accessibility.
